internal/cryptutil/sign.go

```diff
@@ -9,6 +9,11 @@ import (
 	"gopkg.in/square/go-jose.v2/jwt"
 )
 
+const (
+	// DefaultLeeway defines the default leeway for matching NotBefore/Expiry claims.
+	DefaultLeeway = 2.0 * time.Minute
```

Comment:
5 might be a good number here as it builds on Kerberos time sync constraint, which is probably a baked in assumption in other adjacent systems.

Reply:
Thanks, I was going back and forth about this. Really good idea.

Reply:
Fixed!